src/pages/categories/Categories.tsx

```diff
@@ -43,6 +47,22 @@ const Categories = () => {
     setIsFetched(true)
   }
 
+  useEffect(() => {
+    if (match) {
+      const categoryArray = categoriesNamesItems.filter(
+        (category) => category === match
+      )
+      if (categoryArray.length === 0) {
+        setIsCategoryFound(true)
+      } else {
+        setIsCategoryFound(false)
+      }
+    }
+    if (!match) {
+      setIsCategoryFound(false)
+    }
+  }, [isSearchButtonClicked])
```

Comment:
`categoriesNamesItems` and `match` shoud be added to array
```suggestion
  }, [isSearchButtonClicked, categoriesNamesItems, match])
```

Reply:
added